### DISCUSSION TITLE
大家都用什麼APP作筆記

### DISCUSSION THREAD

**Discussion by todoroki0910** [1101-GS3073-C/Python-Community-GS3073-C#530] [General]
我之前是用 onenote但覺得很難用，想問大家都用什麼做筆記

**Comment by gnsJhenJie** [comment]
我用GoodNotes用得很順手

**Comment by brian431** [comment]
GoodNotes +1

**Comment by BingRong1217** [comment]
Notion很好用!!

**Comment by awkward-willy** [comment]
個人已經習慣使用 Notion 了，但近期看到 Obsidian 很紅並有很多功能很特殊又吸引人，不知道有沒有人實際使用過。

**Comment by Ben6uo** [comment]
我都會手寫 但不一定會看就是了

**Comment by williamyen0623** [comment]
一開始用notability，
後來goodnotes特價，
一試成主顧，不管是範圍截圖，資料導入還有很多強大的功能我覺得用起來都比前者好用，
不過我是寫字才會用到goodnotes，
如果是要歸類整理知識，我都是用notion居多，更有系統化的排版用起來比較順手，也比較好看，因為我寫字賊醜

**Comment by HsuChen88** [comment]
goodnote +1

**Reply by HsuChen88** [reply]
這樣也被AI抓出來，Ai果然不好惹

**Comment by skyling0299** [comment]
剛開始是onnote，
後來入手goodnotes，
我喜歡在goodnote上面書寫的感覺，我覺得蠻有意思，移動與選取的功能也很好用

**Comment by RogelioKG** [comment]
一開始在整理高中的電子筆記都是用 Google Keep ，
Google Keep 的優點是方便，
打開手機就可以快速紀錄，
不過 Google Keep 僅限簡單的文字紀錄。
半年前開始學程式之後，
就開始使用 Notion，
Notion 讓一頁筆記有不同的層次結構
（像 /tog 就是收納清單，也有四種不同大小），
這會讓多小節內容的筆記整理起來一目了然。
最重要的是它支援 LaTeX，
能放進帥帥的數學公式就是讚啦！

**Comment by Raywei0214** [comment]
我都用活頁紙，用紅藍黑原子筆跟鉛筆

**Comment by Raywei0214** [comment]
洛克曾經認為，智育只能是德育的輔助品，學問只能作為輔佐品德之用，對於心地良好的人來說，學問對於德行與智慧都有幫助; 對於心地不是良好的人來說，學問就會使他們變得更壞。

**Reply by RogelioKG** [reply]
Woah 點數駭客
這個 AI 應該要偵測一下內文相關性啊

**Reply by chia-kai-chang** [reply]
我來了 👀

**Comment by 710388** [comment]
goodnotes，開始使用平板作筆記後我才發現比紙筆方便了許多，不管是直線、圓形能很美，也不會有墨水沒乾的問題

**Comment by bww0401** [comment]
我都用紙跟自動鉛筆(數學系的習慣...

**Comment by awehao** [comment]
GOODNOTES，雖然有時會造成平板過熱，部過我覺得利大於弊，鎮的是一個很方便的程式

**Comment by magicmicheal** [comment]
Goodnotes！ 開始使用平板作筆記後我才發現比紙筆方便了許多，不會有墨水沒乾的問題，當拿到了老師講議的PDF也能快速地匯入APP中，甚至有微積分以及許多參考書的電子檔後，甚至可以省下不少額外的開銷，要說他的缺點就是平板容易過熱，天氣乾的時候apple pencil握起來不太舒服

**Comment by larry40305** [comment]
原本進大學後有考慮也用平板來做筆記，但後來沒有買，因為發現自己寫筆記好像對自己沒幫助。

**Comment by titan92601** [comment]
自從買了平板就用GoodNote 但是用久了會覺得不太舒服，可能是長時間盯著平板吧 現在我只用GoodNote看課本而已 計算還是活頁紙方便！

**Comment by boson13579** [comment]
goodnote版面美觀、筆記頁面豐富多元，比較適合喜歡看每每筆記的學生。
notability有一個好處是會在頁末幫你添加一頁空白頁，操作也比較簡單好上手，微不足的是他的版面較單調，適合筆記新手。
goodnote和notability兩個都能將pdf導入。

**Comment by Valentinaa0214** [comment]
用goodnote作筆記！身邊的人也都是用goodnote，
之前有嘗試過notability，但寫起來超不順，我也沒有那麼喜歡它的設計，
最後還是覺得goodnote比較順手~

**Comment by 111607004** [comment]
我都用collanote，裡面還有內建錄音和掃描功能，雖然前者我不太常用，但感覺還蠻酷的。
我覺得他的缺點是內建的便利貼插圖非常醜，而且圖案很不實用。
